messaging/src/androidTest/java/io/lantern/messaging/MessagingTest.kt

```diff
@@ -101,6 +101,14 @@ class MessagingTest : BaseMessagingTest() {
                                 cat,
                                 "hi cat"
                             )
+
+                            dog.waitFor<Model.Contact>(
```

Comment:
Drive-by, this makes it easier to identify if we've been contact by someone so we can suppress the message about pending introductions.